bedrock/firefox/views.py

```diff
@@ -372,6 +372,9 @@ def get_context_data(self, **kwargs):
         # add version to context for use in templates
         ctx['version'] = self.kwargs.get('version') or ''
 
+        # add funnelcake to context for use in templates
+        ctx['funnelcake'] = self.request.GET.get('f', '')
```

Comment:
Good catch. This is bad. What's happening with `version` above is different. That comes out of the `kwargs` which are parsed from the URL based on a regex. So it's already safe. Getting anything from `request.GET` and putting it directly into the context is never okay. I think we already have an example of getting funnelcake safely though...